src/QueryBuilder/HasAttributesQueryBuilder.php

```diff
@@ -10,31 +10,43 @@
 
 namespace BitBag\SyliusElasticsearchPlugin\QueryBuilder;
 
+use BitBag\SyliusElasticsearchPlugin\QueryBuilder\AttributesQueryBuilder\AttributesQueryBuilderCollectorInterface;
+use BitBag\SyliusElasticsearchPlugin\Repository\ProductAttributeRepositoryInterface;
 use Elastica\Query\AbstractQuery;
-use Elastica\Query\BoolQuery;
-use Elastica\Query\Term;
 use Sylius\Component\Locale\Context\LocaleContextInterface;
 
 final class HasAttributesQueryBuilder implements QueryBuilderInterface
 {
     private $localeContext;
 
-    public function __construct(LocaleContextInterface $localeContext)
-    {
+    /** @var ProductAttributeRepositoryInterface */
+    private $productAttributeRepository;
+
+    /** @var AttributesQueryBuilderCollectorInterface[] */
+    private $attributeDriver;
+
+    public function __construct(
+        LocaleContextInterface $localeContext,
+        ProductAttributeRepositoryInterface $productAttributeRepository,
+        iterable $attributeDriver
+    ) {
         $this->localeContext = $localeContext;
+        $this->productAttributeRepository = $productAttributeRepository;
+        $this->attributeDriver = $attributeDriver;
     }
 
     public function buildQuery(array $data): ?AbstractQuery
     {
-        $attributeQuery = new BoolQuery();
+        $attributeName = str_replace('attribute_', '', $data['attribute']);
```

Comment:
Are you sure you need it ? I think not 
if you need an alias for attribute name you can use simply `$attributeName = $data['attribute']`